xml/installation-installation-esx-prepare_deploy_esx_cloud_ovsvapp.xml

```diff
@@ -358,7 +358,7 @@ enabled_backends=&lt;unique-section-name&gt;
 #vmware_task_poll_interval = 0.5
 #vmware_max_objects_retrieval = 100
 #vmware_volume_folder = cinder-volumes
-#volume_driver = cinder.volume.drivers.vmware.vmdk.VMwareVcVmdkDriver
+#volume_driver = cinder.volume.drivers.vmware.vmdk.&vmware;VcVmdkDriver
```

Comment:
I think this one should have remained - because it maps to python classes in OpenStack and if someone updates the &vmware; define, it would break the example

Reply:
Good catch, thanks! I will switch this one.